packages/core/src/configuration.ts

```diff
@@ -5,7 +5,6 @@ import { includes, ONE_KILO_BYTE, ONE_SECOND } from './utils'
 
 export const DEFAULT_CONFIGURATION = {
   allowedTracingOrigins: [] as Array<string | RegExp>,
-  isCollectingError: true,
```

Comment:
Just letting you know this is a breaking change but wasn't reported as such in your changelog.

Reply:
👋 @tbillington, it should not be.
There was an issue in the logs documentation, `isCollectingError` was not an init parameter for logs, `forwardErrorToLogs` was the right one: https://github.com/DataDog/browser-sdk/pull/545/files#diff-2e4bca1144de53c8c504edb040ac34aa7ea63ba413a4453277d88307278d8972R43.
It is still collecting error by default: https://github.com/DataDog/browser-sdk/pull/545/files#diff-c62c675364c11388aeae30517afdac97f57f2638e8fb3d019177ebfeb382f540R25

If you still see something wrong, would you mind opening a dedicated issue and provide more details?